.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ default_language_version:
   python: python3
 repos:
 - repo: https://github.com/psf/black
-  rev: 19.3b0
+  rev: 19.10b0
```

Comment:
Please don't change this, as it's pinned by a reason and controlled in a global way. Is there any reason for doing this?

Reply:
Version `19.3b0` is **outdated** and  has a lot of issues with trailing comma  and how to pass arguments to methods which force to go against `PEP8`. 
Trailing comma [example](https://travis-ci.org/OCA/reporting-engine/jobs/629075511#L321)
Readability [example](https://travis-ci.org/OCA/reporting-engine/jobs/629054850#L371)
Trailing comma + readability + arguments [example](https://travis-ci.org/OCA/reporting-engine/jobs/629054850#L437)

I asked before can I make "strange" changes requested by `black` in separate commit with message what these changes are from linter. The answer was **"no, it's a standard. everyone should follow `black` recommendations".** And I made a research and figured out that the relevant version of `black` actually is fine with the code, but not the one which in use in this repository.

Reply:
Uhm, not sure to understand your answer, sorry.

@sbidoul did you get what @Tatider is commenting? Should we globally rise black version?

Reply:
@pedrobaeza there is indeed a black version with (relatively minor IMO) improvements. 

I would recommend to keep the black version the same for all OCA repos so contributors don't need to switch version in their IDE when working on different repos.

That said, I'm still planning to push the `.pre-commit` config update to all repos (including the latest black version) soon.

Reply:
Commit with `black` version upgrade was removed